public/ge0.html

```diff
@@ -76,21 +86,19 @@
 </head>
 
 <body>
-  <div id="header">
-    <span id="name">${name}</span>
-    <span>
-      <a id="open" href="om:/${path}" class="link">
```

Comment:
No, we can not remove om:// link. It is the only way to open https://omaps.app links on many androids and iOS devices where it's not open by default from the app.